lib/victoria_metrics/client.rb

```diff
@@ -7,14 +7,18 @@
 require "zlib"
 
 class VictoriaMetrics::Client
-  def initialize(endpoint:, ssl_ca_file_data: nil, socket: nil, username: nil, password: nil)
+  def initialize(endpoint:, ssl_ca_data: nil, socket: nil, username: nil, password: nil)
     @endpoint = endpoint
     @username = username
     @password = password
-    @client = if ssl_ca_file_data
-      ca_bundle_filename = File.join(Dir.pwd, "var", "ca_bundles", Digest::SHA256.hexdigest(ssl_ca_file_data) + ".crt")
-      Util.safe_write_to_file(ca_bundle_filename, ssl_ca_file_data) unless File.exist?(ca_bundle_filename)
-      Excon.new(endpoint, socket: socket, ssl_ca_file: ca_bundle_filename)
+    @client = if ssl_ca_data
+      cert_store = OpenSSL::X509::Store.new
+      certs_pem = ssl_ca_data.scan(/-----BEGIN CERTIFICATE-----.*?-----END CERTIFICATE-----/m)
+      certs_pem.each do |cert_pem|
```

Comment:
Could we do `certs = OpenSSL::X509::Certificate.load_file(ca_bundle_filename)` here instead of parsing the file via regex ourselves?